rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/fleet_manager.py

```diff
@@ -342,10 +366,10 @@ async def start_activity(
                 target_loc.level_name = map_name
                 path_request.path.append(target_loc)
 
-            path_request.fleet_name = self.fleet_name
-            path_request.robot_name = robot_name
-            path_request.task_id = str(cmd_id)
-            self.path_pub.publish(path_request)
+                path_request.fleet_name = self.fleet_name
+                path_request.robot_name = robot_name
+                path_request.task_id = str(cmd_id)
+                self.path_pub.publish(path_request)
 
             if self.debug:
```

Comment:
This doesn't quite seem right, since spaces are significant in Python this would do a bunch of unnecessary assignments and especially publish a message for point in the path array

Reply:
Thanks for catching that, reverted in [ce846a1](https://github.com/open-rmf/rmf_demos/pull/225/commits/ce846a1ca15fdac7b01667d6564c1bdd42225852)